localstack/services/awslambda/lambda_api.py

```diff
@@ -1134,7 +1143,9 @@ def create_event_source_mapping():
               in: body
     """
     data = json.loads(to_str(request.data))
-    mapping = add_event_source(data['FunctionName'], data['EventSourceArn'], data.get('Enabled'))
+    mapping = add_event_source(
+        data['FunctionName'], data['EventSourceArn'], data.get('Enabled'), data.get('BatchSize', 10)
```

Comment:
nit: perhaps we can remove the default value (`10`) here and handle that case in `add_event_source` (see comment above).
```

    mapping = add_event_source(
        data['FunctionName'], data['EventSourceArn'], data.get('Enabled'), data.get('BatchSize') 
    )
```